### PR TITLE
Components

### PR DESCRIPTION
**Pull request by UsirusenIkon**
In this project, we designed the foundations for our Bookstore website. We created the React part of the app. We also set up routing using React Router.

### CODE REVIEW

File: src/components/Books/Form/Form.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+          <label for="book name">
+            <input
+              name="fullname"
+              class="wf"
+              type="text"
+              placeholder="Book Title"
+              maxlength="30"
+              id="book-name"
+              required
+            />
```
Comment (EricMbouwe) [inline src/components/Books/Form/Form.jsx]: - [OPTIONAL] The value of the label's attribute `for` should match the value of the input's `id` to make a correct relation between them.

File: src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <div className="App">
+      <nav>
+        <h1>Bookstore CMS</h1>
+        <ul>
+          <li><Link to="/">Books</Link></li>
+          <li><Link to="/Categories">Categories</Link></li>
+        </ul>
+      </nav>
+      <Routes>
+        <Route path="/" element={<Books />} />
```
Comment (EricMbouwe) [inline src/App.jsx]: - [ ] In order to not show a blank page when visiting the site for the first time, the BookList component should be the default page. To make it functional you should use the `Route exact` prop.

eg: 
```jsx
<Route exact path="/">
      <BookList />
</Route>
```

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @UsirusenIkon, 

Good job so far 🔥!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🔥🎉
- Add 2 `<Link>` in the navbar ✔
- Add 2 `<Route>` for the BookList component and the other for the Categories component ✔
- Add components ✔
- No linter errors ✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @UsirusenIkon 👋, 

Well done for implementing the required changes 👏 
Your project is now complete and there is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**